layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard-4.4/0001-omap_hsmmc-Reduce-max_segs.patch

```diff
@@ -0,0 +1,36 @@
+From fb8ec188334f571e835115a0deb7b5be8662bb38 Mon Sep 17 00:00:00 2001
+From: Will Newton <willn@resin.io>
+Date: Fri, 16 Jun 2017 12:13:21 +0200
+Subject: [PATCH] omap_hsmmc: Reduce max_segs
+
+Reduce max_segs to a value that allows allocation of an entire
+descriptor list within a single page. This avoids doing a
+higher order GFP_ATOMIC allocation when setting up a transfer
+which can potentially fail and lead to I/O failures.
+
+Upstream-Status: Pending
```

Comment:
The relevant patch (that I am aware of) is:
```
commit f048f8dd75c9cd7742c4d0f9ae549e18640ef58e
Author: Petros Angelatos <petrosagg@gmail.com>
Date:   Tue Dec 13 12:28:17 2016 -0800

    fix-mmc-bbb: Memory tuning to prevent higher-order allocation failures
    
    Signed-off-by: Petros Angelatos <petrosagg@gmail.com>

diff --git a/layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf b/layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf
index 788f3be..fb9aa95 100644
--- a/layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf
+++ b/layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf
@@ -1 +1,3 @@
-vm.vfs_cache_pressure=400
+vm.min_free_kbytes=8192
+vm.dirty_ratio=5
+vm.dirty_background_ratio=10
```

Which is present in 1.26-rc0.